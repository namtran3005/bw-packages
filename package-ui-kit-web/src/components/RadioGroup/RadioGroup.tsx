import * as React from 'react';
import { RadioGroup as MuiRadioGroup } from '@material-ui/core';
import { RadioGroupProps as MuiRadioGroupProps } from '@material-ui/core/RadioGroup';

const RadioGroup: React.FunctionComponent<MuiRadioGroupProps> = (props) => (
  <MuiRadioGroup {...props} />
);

RadioGroup.displayName = 'RadioGroup';

export { RadioGroup, MuiRadioGroupProps as RadioGroupProps };
