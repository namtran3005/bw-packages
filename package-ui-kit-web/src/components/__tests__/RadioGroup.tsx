import * as React from 'react';
import { shallow } from 'enzyme';
import { RadioGroup } from '../RadioGroup';

const defaultProps = {};

describe('RadioGroup', () => {
  it('should render radio group', () => {
    const wrapper = shallow(<RadioGroup {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
