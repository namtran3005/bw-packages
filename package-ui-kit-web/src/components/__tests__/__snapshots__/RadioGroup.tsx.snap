// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup should render radio group 1`] = `<ForwardRef(RadioGroup) />`;
